packages/coinstac-api-server/src/auth-helpers.js

```diff
@@ -63,6 +77,32 @@ const helperFunctions = {
 
     return result.ops[0];
   },
+  /**
+   * Save password reset token
+   * @param {string} email email to send password reset token
+   * @return {object}
+   */
+  async savePasswordResetToken(email) {
+    const resetToken = helperFunctions.createPasswordResetToken(email);
+
+    const msg = {
+      to: email,
+      from: 'no-reply@mrn.org',
+      subject: 'Password Reset Request',
+      html: `We received your password reset request. <br/>
+        Please use this token for password reset. <br/>
+        Token: <strong>${resetToken}</strong>`,
+    };
+
+    const db = database.getDbInstance();
+
+    await sgMail.send(msg);
+    await db.collection('users').findOneAndUpdate({ email }, {
```

Comment:
Since you do not want to get the user after the operation you should use updateOne. Also, you don't need to wait for the operation... just return it.